pandas/core/reshape/reshape.py

```diff
@@ -317,14 +318,14 @@ def get_new_columns(self, value_columns: Index | None):
         width = len(value_columns)
         propagator = np.repeat(np.arange(width), stride)
         if isinstance(value_columns, MultiIndex):
-            new_levels = value_columns.levels + (self.removed_level_full,)
+            new_levels: FrozenList | list[Index] = value_columns.levels + (
+                self.removed_level_full,
+            )
```

Comment:
really minor, but this has become a bit harder to read, how about
```suggestion
            new_levels: FrozenList | list[Index]
            new_levels = value_columns.levels + (self.removed_level_full,)
```
? (feel free to ignore if you disagree though)

Reply:
I agree. Would even move it in front of the if condition